l5kit/l5kit/geometry/transform.py

```diff
@@ -72,32 +72,61 @@ def flip_y_axis(tm: np.ndarray, y_dim_size: int) -> np.ndarray:
     return tm
 
 
-def transform_points(points: np.ndarray, transf_matrix: np.ndarray) -> np.ndarray:
+def transform_points_batch(points_batch: np.ndarray, transf_matrix: np.ndarray) -> np.ndarray:
```

Comment:
I think that `batch` is too overloaded.

in fact the original `transform_points` is handling 

`(a time series of points on a trajectory, 2D or 3D points)`
or
`(N, 2/3)`

but now `transform_points_batch` is

`(num of trajectories, a time series of points on a trajectory, 2D or 3D points)`
or
`(T, N, 2/3)`

Is this what you meant (or, more precisely, want)?

Reply:
Having:
`(T, N, 2/3)` won't make a difference here, because you can always view it as `(T*N, 2/3)`.

The issue people are facing is that you have a set of trajectories and a set of transformation matrices (one per trajectory). What we should provide is a way of doing that with a single function call.

Reply:
i guess in that case i can close this until when i can think of a more solid usecase.

Reply:
FYI: I'll start the PR for supporting batches in both args later this week, I'll ping you there if it's fine with you